pom.xml

```diff
@@ -128,6 +128,11 @@
         </dependency>
 
         <!-- Runtime dependencies -->
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jsonSchema</artifactId>
+            <version>2.15.2</version>
```

Comment:
Put the version in dependencyManagement

Reply:
Oh alright, my bad I wasn't aware of this practice